core/aws.py

```diff
@@ -0,0 +1,36 @@
+from io import FileIO
+
+from boto3 import client
+from botocore.exceptions import ClientError
+from flask import current_app
+
+from config import Config
+
+if Config.AWS_ACCESS_KEY_ID and Config.AWS_SECRET_ACCESS_KEY:
+    _S3_CLIENT = client(
+        "s3",
+        aws_access_key_id=Config.AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=Config.AWS_SECRET_ACCESS_KEY,
+        region_name=Config.AWS_REGION,
+        endpoint_url=Config.AWS_ENDPOINT_OVERRIDE,
+    )
```

Comment:
Can we remove this block? We should never be passing access keys and secret keys in on AWS.